### PR TITLE
Moved Markdown files to doc directory

### PR DESCRIPTION
**Pull request by shaswat-satyam**
Issue #304 

### Changes Proposed
- Moved Markdown Files to a separate doc Directory

### CODE REVIEW

File: docs/README.md
Comment (PSNAppz) [inline docs/README.md:1]: Don't move the README.md. It has to be in the root dir.

**Review by PSNAppz — CHANGES_REQUESTED** [review]

**Review by KingAkeem — CHANGES_REQUESTED** [review]
Could you remove the pull request template? It's being moved in another PR, https://github.com/DedSecInside/TorBot/pull/306.

**Comment by shaswat-satyam** [comment]
Done

**Review by PSNAppz — CHANGES_REQUESTED** [review]
One last thing. We are referencing the `Licensing` url in `Readme`. Can you fix that also?

**Comment by shaswat-satyam** [comment]
There you go!

**Review by KingAkeem — CHANGES_REQUESTED** [review]
Could you move the LICENSE.md back to the root directory so that it can be detected GitHub?

**Review by KingAkeem — APPROVED** [review]

**Pull request merged**